src/unix/getaddrinfo.c

```diff
@@ -200,3 +200,22 @@ void uv_freeaddrinfo(struct addrinfo* ai) {
   if (ai)
     freeaddrinfo(ai);
 }
+
+
+int uv_if_indextoname(unsigned int ifindex, char* ifname, size_t *sizep) {
+  char ifname_buf[UV_IF_NAMESIZE];
+  if (NULL == if_indextoname(ifindex, ifname_buf)) {
```

Comment:
@pekkanikander  you're right, I was overthinking. I was looking into the [musl](https://github.com/davidlazar/musl/blob/master/src/network/if_indextoname.c) implementation and saw it was returning the result of `strncpy` and thought that it *could* happen that the buffer would end up not NULL terminated in some cases. But that's not possible as `ifreq.ifr_name` is of size `IFNAMSIZ`. Sorry for the noise.